dali/test/python/test_as_dlpack.py

```diff
@@ -0,0 +1,76 @@
+from nvidia.dali.pipeline import Pipeline
+import nvidia.dali.ops as ops
+import nvidia.dali.types as types
+import torch.utils.dlpack as torch_dl
+import numpy as np
+import os
+import random
+import cupy
```

Comment:
Please also add loading to MXNet in the test.

Reply:
Scope of this task was reduced and it no longer implements transferring data to other frameworks.